### ISSUE TITLE
There is no way to access an AuthAppRoutes instance

### ISSUE THREAD

**Issue by lluismontero** [dropbox/dropbox-sdk-dotnet#39]
As far as I can see, there is no way to access an instance of the **AuthAppRoutes** class (wich contains the methods for migrating from old OAuth1 tokens to the new OAuth2 token) and there isn't any public constructor available.

Is it possible to provide a new property on the **DropboxClient** class to access those methods?

**Comment by lluismontero** [comment]
I've cloned the repo and updated the DropboxClient to provide an AuthAppRoutes property but the request to "/auth/token/from_oauth1" is not work as all the SDK is prepared to send a "Bearer" authentication header but this services requires a "Basic" authentication. Are you planing to support this on a near future?

**Comment by greg-db** [comment]
Thanks for the report! We'll look into it.

**Comment by greg-db** [comment]
This is fixed now, as of [release v4.1.0](https://github.com/dropbox/dropbox-sdk-dotnet/releases/tag/v4.1.0).

You can now do something like this:
```
var client = new DropboxAppClient ("APP_KEY", "APP_SECRET");
var result = await client.Auth.TokenFromOauth1Async ("OAUTH1_ACCESS_TOKEN_KEY", "OAUTH1_ACCESS_TOKEN_SECRET");
Console.WriteLine(result.Oauth2Token);
```

**Comment by lluismontero** [comment]
Good (and really fast) work @greg-db !
